src/test/test-sizeof.c

```diff
@@ -93,5 +93,13 @@ int main(void) {
 
         printf("timeval: %zu\n", sizeof(struct timeval));
         printf("timespec: %zu\n", sizeof(struct timespec));
+
+        void *x = malloc(100);
+
+        printf("local variable: %p\n", &function_pointer);
+        printf("glibc function: %p\n", memcpy);
+        printf("heap allocation: %p\n", x);
```

Comment:
if malloc fails, will printf complain that x is null?

Reply:
It's just printing the pointer value, and 0 should be a completely valid value. FWIW, glibc prints `(nil)`, I don't know if this is standarized somehow.